hazelcast/src/main/resources/cluster.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
    	echo "parameters : "
-   	echo "	-o, --operation	    : Defines state operation of the script. Operation can be 'get' or 'change'."
+   	echo "	-o, --operation	    : Defines state operation of the script. Operation can be 'get','change','shutdown','force-start'."
```

Comment:
`get-state`,`change-state` would be better names for those operations.

Another naming suggestion :
`Defines state operation of the script` --> `executes cluster-wide operation`